travis-publish.sh

```diff
@@ -9,7 +9,7 @@ export TAG=`if [ "$TRAVIS_PULL_REQUEST" = "false" -a -n "$TRAVIS_TAG" ] ; then e
 
 if [ "$TAG" ]; then
   echo "Build is tagged. Uploading artifact $TAG to Bintray."
-  ./gradlew --info -Dbuild.version=$TRAVIS_TAG bintrayUpload
+  ./gradlew --info -Dbuild.version="$TAG" gitPublishPush bintrayUpload
```

Comment:
@andrii-kovalenko-ct Question: if the `gitPublishPush` fails due to authentication failure (or any other reason) will the `bintrayUpload` task still be executed? and also could it happen that the `gitPublishPush` succeeds but the `bintrayUpload` fails? 

We should ensure that if either of them fails, both are not excuted.

Reply:
6c99781b
 -  if `gitPublishPush` fails - `bintrayUpload` won't start, the build fails
 -  if `gitPublishPush` success, but `bintrayUpload` fails - print warning message and the build fails
 - if both success - the build success